### DISCUSSION TITLE
Toasts! (MRTK3)

### DISCUSSION THREAD

**Discussion by camnewnham** [microsoft/MixedRealityToolkit-Unity#11091] [Ideas]
Hello :)
At the moment we use native Windows Toast elements to show notifications of various in-app events (as well as using WNS, but that's a separate story) - this is handy because like Windows dialogs (as separate to MRTK dialogs) they render over all other content and are always consistent.  With MRTK2 this works reasonably well, as the MRTK style matches the HoloLens OS style. 

The downside is they are styled with the classic Windows shell and are not themed like MRTK3. Is there any intention/interest to add a toast element? The API would presumably be quite similar to `Dialog.Show` and the [layouts are well defined](https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/adaptive-interactive-toasts?tabs=builder-syntax) (though presumably not all of these would be implemented!) .

**Comment by Zee2** [comment]
Good idea! We're drowning in work a bit right now but this would be nice to have in the future for sure.
